canonical-web-and-design/maas-ui
# Table loading state

Done:
- Update settings and preferences lists to use a shared table component.
- Add new loading state. Fixes: #159.

QA:
- Click through all the lists in settings and preferences.
- There should be a new loading state (see: #159).
- Everything else should function as it did before.
